src/Calculator/Views/GraphingCalculator/EquationInputArea.xaml.cpp

```diff
@@ -80,6 +80,8 @@ void EquationInputArea::AddNewEquation()
 
 void EquationInputArea::InputTextBox_GotFocus(Object ^ sender, RoutedEventArgs ^ e)
```

Comment:
I think you're right.  I'll try having the equation text box keep track of it's last submitted equation.

Reply:
Rudy already did the de-duping logic (Thanks Rudy!), so now I can just ensure that all submit paths go through InputTextBox_Submitted and it will only re-render when needed.  Here all I needed to do was always call submit when key graph features is requested